application_store_api/namespace_fund/__init__.py

```diff
@@ -0,0 +1 @@
+from application_store_api.namespace_fund.routes import fund_ns
```

Comment:
Just wondering about the file structure of this for future development - might `namespaces/fund/` be an alternative? Or, if everything under `application_store_api` is a namespace then could have just `fund/`? Also, seeing as this is the application_store service perhaps `application_store_api` could be reduced to just `api`?